delta_time_secs : 0.2

usv_params : {
  usv_1 : {
    sim_id : 1,
    radius_buffer : 50,
    init_state : {
        x : 50,
        y : 50,
        heading : 30,
        speed : 0
      },
    constraints : {
      max_speed : 50,
      max_delta_speed : 100,
      max_delta_heading : 90
      },
    radar_parameters : {
      max_distance : 60,
      max_angle : 60,
      aggression : 0.7
      } 
    },
    
  usv_2 : {
    sim_id : 2,
    radius_buffer : 50,
    init_state : {
        x : -50,
        y : 50,
        heading : -30,
        speed : 0
        },
    constraints : {
      max_speed : 50,
      max_delta_speed : 100,
      max_delta_heading : 90
      },
    radar_parameters : {
      max_distance : 60,
      max_angle : 60,
      aggression : 0.7
      }
    },
  usv_3 : {
    sim_id : 3,
    radius_buffer : 50,
    init_state : {
        x : 50,
        y : -50,
        heading : 30,
        speed : 0
      },
    constraints : {
      max_speed : 50,
      max_delta_speed : 100,
      max_delta_heading : 90
      },
    radar_parameters : {
      max_distance : 60,
      max_angle : 60,
      aggression : 0.7
      } 
    },
    
  usv_4 : {
    sim_id : 4,
    radius_buffer : 50,
    init_state : {
        x : -50,
        y : -50,
        heading : -30,
        speed : 0
        },
    constraints : {
      max_speed : 50,
      max_delta_speed : 100,
      max_delta_heading : 90
      },
    radar_parameters : {
      max_distance : 60,
      max_angle : 60,
      aggression : 0.7
      }
    }
  }
intruder_params : {
  intruder_101 : {
    sim_id : 101,
    radius_buffer : 50,
    init_state : {
        x : 800,
        y : 0,
        heading : 30,
        speed : 0
      },
    constraints : {
      max_speed : 30,
      max_delta_speed : 100,
      max_delta_heading : 45
      },
    radar_parameters : {
      max_distance : 100,
      max_angle : 60,
      aggression : 0.1
      } 
    }
    
  ,intruder_102 : {
    sim_id : 102,
    radius_buffer : 50,
    init_state : {
        x : -800,
        y : 0,
        heading : -30,
        speed : 0
        },
    constraints : {
      max_speed : 50,
      max_delta_speed : 100,
      max_delta_heading : 45
      },
    radar_parameters : {
      max_distance : 100,
      max_angle : 60,
      aggression : 0.1
      }
    }
  }

asset_params : {
  asset : {
    radius_buffer : 60,
    sim_id : 100,
    init_state : {
        x : 0,
        y : 0,
        heading : 0,
        speed : 0
      }
  }
}